reflectable/lib/src/builder_implementation.dart

```diff
@@ -5005,7 +5005,18 @@ Future<String> _extractMetadataCode(Element element, Resolver resolver,
       if (_isPrivateName(name)) {
         await _severe('Cannot access private name $name', element);
       }
-      metadataParts.add('const $prefix$name($arguments)');
+      if (annotationNode.typeArguments != null) {
```

Comment:
We usually use a style (maybe it isn't consistent in this library, but we're trying to be consistent at least with code that gets updated ;-) where null is handled. So we'd have something like this:

```suggestion
      var typeArguments = annotationNode.typeArguments;
      if (typeArguments != null) {
```

This means that the local variable `typeArguments` is promoted to have type `TypeArgumentList` rather than `TypeArgumentList?`, and then we don't have to test for null when it's used later.